test/src/test/java/hudson/cli/EnableJobCommandTest.java

```diff
@@ -21,36 +21,27 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package hudson.cli
 
-import static org.junit.Assert.assertFalse
-import static org.junit.Assert.assertTrue
+package hudson.cli;
 
-import org.junit.Rule
-import org.junit.Test
-import org.jvnet.hudson.test.JenkinsRule
+import hudson.model.FreeStyleProject;
+import static org.junit.Assert.*;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
 
-/**
- * @author Kohsuke Kawaguchi
- */
 public class EnableJobCommandTest {
 
     @Rule
-    public JenkinsRule j = new JenkinsRule()
+    public JenkinsRule r = new JenkinsRule();
 
     @Test
-    void test() {
-        def p = j.createFreeStyleProject()
-
-        def cli = new CLI(j.getURL())
```

Comment:
One of a few forgotten in #2795 because they were in Groovy code and so did not appear in my usage search.